$NetBSD: patch-bj,v 1.2 2010/09/27 12:01:49 taca Exp $

* Make it build on Linux by conditionally removing conflicting declarations.

--- h/strings.h.orig	2001-04-02 10:35:24.000000000 +0000
+++ h/strings.h
@@ -35,7 +35,7 @@
 char   *index ();
 char   *mktemp ();
 char   *rindex ();
-#ifndef	SPRINTFTYPE
+#if !defined(SPRINTFTYPE) && !defined(linux)
 #ifndef	ncr		/* NCR compiler complains about re-declaration */
 char   *sprintf ();		/* I guess this is the new standard */
 #endif
@@ -47,7 +47,7 @@ SPRINTFTYPE sprintf ();
 char   *strcat ();
 int     strcmp ();
 char   *strcpy ();
-#ifndef hpux
+#if !defined(hpux) && !defined(linux)
 int	strlen ();
 #endif
 char   *strncat ();
@@ -55,7 +55,7 @@ int     strncmp ();
 char   *strncpy ();
 #endif
 
-#if !defined(SVR4) && !defined(__386BSD__) && !defined(BSD44)
+#if !defined(SVR4) && !defined(__386BSD__) && !defined(BSD44) && !defined(linux)
 char   *getenv ();
 char   *calloc (), *malloc (), *realloc ();
 #endif	/* SVR4 */
